azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -133,8 +144,38 @@ func resourceArmSpringCloudService() *schema.Resource {
 				},
 			},
 
+			"trace": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
+							Required: true,
```

Comment:
rather than having `enabled` as a field here - we can use the presence of this block to assume `enabled = true` and the ommission of this block to mean `enabled=false` - so we can remove this field